spec/requests/social_accounts_request_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'SocialAccounts', driver: :selenium_headless, type: :request do
+  let(:user) { FactoryBot.create :user }
+
+  before do
+    sign_in user
+  end
+
+  describe 'POST /users/:user_id/social_accounts' do
+    context 'with valid parameters' do
+      it 'creates a new social account' do
+        post user_social_accounts_path(user), params: { social_account: { social_network: 'GitHub', user_name: 'Foo' } }
+        expect(response).to redirect_to(edit_user_url(user))
+        expect(user.social_accounts.count).to eq(1)
+      end
+
+      it 'creates a new social account with invalid attributes' do
```

Comment:
still the wrong description, isn't it? I would expect something like `does not create a new social account with invalid attributes`